src/urbit/app/chess.hoon

```diff
@@ -812,18 +822,19 @@
           ==
```

Comment:
The card on lines `816` - `819` should be removed. If a sent challenge gets nacked, we don't need to send an update card to the frontend, since we have nothing to remove from the frontend state, since we never sent a card to add this challenge to the frontend.